src/jubilation-line/index.jsx

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from 'react';
+import getContext from '../helpers/context';
+import Animation from '../jubilation-animation';
+
+type Props = { x1?: number, x2?: number, y1?: number, y2?: number};
+type Context = { JubilationContext: JubilationContext };
+
+/**
+ * A line segment, defined by two endpoints. Returns an SVG line element.
+ */
+export default function JubilationLine({ x1, y1, x2, y2 }: Props,
```

Comment:
lets spread the rest of the props so we can apply them to the rendered line
I think this will work, but I am not totally sure
`{ x1, y1, x2, y2, ...props }`